js/rpg_windows/Window_MenuStatus.js

```diff
@@ -43,7 +43,7 @@ Window_MenuStatus.prototype.numVisibleRows = function() {
 
 Window_MenuStatus.prototype.loadImages = function() {
     $gameParty.members().forEach(function(actor) {
-        ImageManager.loadFace(actor.faceName());
+        ImageManager.reserveFace(actor.faceName());
```

Comment:
If It reserves, It must be guarded by `ImageManager.isReady()` and It breaks original behavior(chance to show NowLoading).

And, Original source is already lack of independence about image resource.